openshift/release/knative-eventing-ci.yaml

```diff
@@ -4184,6 +4184,7 @@ spec:
           image: registry.svc.ci.openshift.org/openshift/knative-v0.18.4:knative-eventing-mtping
           env:
             - name: SYSTEM_NAMESPACE
+              value: ''
```

Comment:
?!

Reply:
value or value from?

Reply:
We need both: https://github.com/knative/eventing/pull/4472